apps/emqx_plugins/src/emqx_plugins.erl

```diff
@@ -433,9 +433,18 @@ do_ensure_started(NameVsn) ->
     tryit(
         "start_plugins",
         fun() ->
-            ok = ensure_exists_and_installed(NameVsn),
-            Plugin = do_read_plugin(NameVsn),
-            ok = load_code_start_apps(NameVsn, Plugin)
+            case ensure_exists_and_installed(NameVsn) of
+                ok ->
+                    Plugin = do_read_plugin(NameVsn),
+                    ok = load_code_start_apps(NameVsn, Plugin);
+                {error, plugin_not_found} ->
+                    ?SLOG(error, #{
+                        msg => "plugin_not_found",
+                        name_vsn => NameVsn
```

Comment:
The user may have removed the plugin from disk without changing the `data/cluster.config`. This prevents the emqx gets started.